nodebuilder/settings.go

```diff
@@ -125,8 +175,7 @@ func initializeMetrics(
 			semconv.SchemaURL,
 			semconv.ServiceNamespaceKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
 			semconv.ServiceNameKey.String(fmt.Sprintf("semver-%s", buildInfo.SemanticVersion)),
-			semconv.ServiceInstanceIDKey.String(peerID.String()),
-		)))
+			semconv.ServiceInstanceIDKey.String(fmt.Sprintf("%s/%s", network.String(), peerID.String())))))
```

Comment:
let's make the separator a `:` -- doesn't peerID.String() output the full multiaddr or only the key ?

Reply:
I'd leave `/` as it's more visible for eyes than `:`